source/developers-guide/event-guide/index.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: default
+title: Event guide
+github_link: developers-guide/event-guide/index.md
+indexed: true
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+This guide will lead you through the Shopware event system. You can use this as a reference for the different event
+types and you can find useful tips how to find the event you want to use.
+
+## What is an event system?
+An event system will be used to extend an application with your code or reacting on an action which was triggered by
+an user. For example you want to change the behaviour of a button, in that case you can use an event which will be
+fired every time the button was clicked. 
+
+## Example
+To get a better idea of how events are working we will take a look at a example.
+
+For our use case we will register on a pseudo event and try to cancel the application execution.
+
+### Definition of an event
```

Comment:
This is a beginner's document. You are discussing advanced and unneeded topics (event registration) even before you introduced what is really vital and crucial here: event listening.